### DISCUSSION TITLE
Can't see TLS traffic as a flow but visible in wireshark/TCPView

### DISCUSSION THREAD

**Discussion by ross-weir** [mitmproxy/mitmproxy#6026] [Q&A]
I have a problem where I'm intercepting traffic that is visible in wireshark and I can see in TCPView that it's being sent via mitmproxy but it is not appearing as a flow in mitmproxy or mitmweb.

I'm using the experimental `osproxy` mode, it seems to be working fine as in the application continues to work and I can see traffic being routed via the proxy in other tools just not in mitmproxy.

I am running the following command:

```
mitmweb -m osproxy --allow-hosts target.domain.com:1211 --ssl-insecure -v
```

I'm not sure if it's related to the new mode or if I'm doing something wrong. I've tried many variations of the command but can't get it to show traffic in `mitmproxy`.

App details:
Windows 10
I'm using `osproxy` specifically because the app avoids the system proxy otherwise. The app establishes a TLSv1.3 socket connection and then sends custom protocol packets.

Thank you

**Comment by ross-weir** [comment]
I have also tried the `tcp-hosts` option along with many other options with no luck.

Stepped through mitmproxy with a debugger and I can see the connection getting established to the target & messages sent/received, by all accounts everything seems to be working aside from seeing the traffic in any mitmproxy uis

**Comment by mhils** [comment]
Do you see the traffic if you don't pass `--allow-hosts`?
